tests/chainer_tests/links_tests/connection_tests/test_convolution_2d.py

```diff
@@ -17,7 +17,8 @@
     'W_dtype': [numpy.float16, numpy.float32, numpy.float64],
 }))
 @testing.inject_backend_tests(
-    None,
+    ['test_forward', 'test_backward',
+     'test_pickling'],
```

Comment:
Why is this fix needed?

Reply:
it was running double_backward which the original test didn't support.
Probably a cause of increased memory consumption

Reply:
How about setting `skip_double_backward_test = True`?

Reply:
didnt know about that one! 
Much better :D thanks!